foxpuppet/windows/browser/window.py

```diff
@@ -12,7 +12,8 @@
 
 class BrowserWindow(BaseWindow):
 
-    """Representation of a browser window."""
+    """Representation of a browser window.
```

Comment:
We need to include the constructor parameters in the docs. I'm guessing that this docstring is taken priority, so we could either look into adding the docstring to the `__init__` method, or just duplicate the parameters in this docstring.